core/controllers/question_editor.py

```diff
@@ -61,10 +61,10 @@ def post(self):
                 (question_dict['id'] is not None) or
                 ('question_state_data' not in question_dict) or
                 ('language_code' not in question_dict) or
-                (question_dict['version'] != 1)):
+                (question_dict['version'] != 0)):
             raise self.InvalidInputException(
                 'Question Data should contain id, state data, language code, ' +
-                'and its version should be set as 1')
+                'and its version should be set as 0')
```

Comment:
Where is the backend test for this (I would expect to see something that passes a dict with version 1 and it failing)?

Reply:
Version itself wasn't tested before, but this exception is tested, will add another test.

Reply:
Done